src/components/searchStudent/SearchStudent.tsx

```diff
@@ -0,0 +1,58 @@
+
+import './SearchStudent.css';
+import Navbar from '../navbar/Navbar';
+import SearchBar from '../SearchBar';
+
+const SearchStudent = () => {
+  return (
+    <div>
+        <Navbar />
+<div className='main'>
+    <div className="search-student">
+        <h1>إبحث عن طالب</h1>
```

Comment:
for the form you can use this package 
https://react-hook-form.com/
with 
https://zod.dev/
for validation. 
We can do a session about this